src/main/rx/ghst.c

```diff
@@ -293,16 +294,21 @@ STATIC_UNIT_TESTED float ghstReadRawRC(const rxRuntimeState_t *rxRuntimeState, u
 {
     UNUSED(rxRuntimeState);
 
-    // derived from original SBus scaling, with slight correction for offset (now symmetrical
-    // around OpenTx 0 value)
-    // scaling is:
-    //      OpenTx   RC     PWM
-    // min  -1024    172    988us
-    // ctr  0        992    1500us
-    // max  1024     1811   2012us
+    // Scaling 12bit channels (8bit channels in brackets)
+    //      OpenTx   RC         PWM
+    // min  -1024    1024(  0)   988us
+    // ctr  0        2048(128)  1500us
+    // max  1024     3072(255)  2012us
     //
 
-    return (5 * ((float)ghstChannelData[chan] + 1) / 8) + 880;
+    float pwm = 0.0;
+    if(chan < 4) {
```

Comment:
```suggestion
    if (chan < 4) {
```
Did I miss this before or is this new code?